ledger-service/http-json/src/main/scala/com/digitalasset/http/LedgerClientJwt.scala

```diff
@@ -253,9 +254,9 @@ object LedgerClientJwt {
     private[http] object StatusEnvelope {
       def unapply(t: Throwable): Option[Status] = t match {
         case NonFatal(t) =>
-          val s = Status fromThrowable t
+          val s = StatusProto fromThrowable t
```

Comment:
There are two namespaces containing Status: `io.grpc.Status` and `com.google.rpc.Status`
The latter was what we wanted to be able to unpack the error details too.